agent/util-scripts/pbench-collect-sysinfo

```diff
@@ -52,13 +53,38 @@ while true; do
 			shift;
 		fi
 		;;
+		--sysinfo)
+		shift;
+		if [ -n "$1" ]; then
+			sysinfo="$1"
+			shift;
+		fi
+		;;
+		--check)
+		for item in ${sysinfo//,/ }; do
```

Comment:
Pondering this, we'd have a problem if the `--check` was specified before the `--sysinfo` parameter.  Can we have `--check` just set a boolean, and then after processing all the arguments, do that loop check?